pkgs/development/tools/database/squirrel-sql/default.nix

```diff
@@ -14,20 +14,25 @@ in stdenv.mkDerivation rec {
     sha256 = "0b16l7p7klagxnwkx2az4mbyd35kv4aj8xxbwm27pp3spz9dk8m0";
   };
 
-  buildInputs = [
-    jre makeWrapper stdenv unzip
-  ];
+  nativeBuildInputs = [ makeWrapper unzip ];
+  buildInputs = [ jre stdenv ];
```

Comment:
having `stdenv` there does not serve a purpose.

Reply:
Indeed. Will remove.